ui/src/components/CachePurgesTable/index.jsx

```diff
@@ -36,16 +42,21 @@ export default class CachePurgesTable extends Component {
   };
 
   createSortedCachePurgesConnection = memoize(
-    (cachePurgesConnection, sortBy, sortDirection) => {
+    (cachePurgesConnection, sortBy, sortDirection, searchTerm) => {
       const sortByProperty = camelCase(sortBy);
+      const filteredCache = searchTerm
+        ? cachePurgesConnection.edges.filter(({ node }) =>
+            node.cacheName.includes(searchTerm)
+          )
+        : cachePurgesConnection;
 
-      if (!sortBy) {
+      if (!searchTerm) {
         return cachePurgesConnection;
       }
```

Comment:
I just noticed that table sorting doesn't work anymore. Let's fix that.

Reply:
Yes, I figured it out now :)